requirements.txt

```diff
@@ -1,5 +1,5 @@
 numpy
-tensorflow-gpu
+tensorflow-gpu==2.1.0
```

Comment:
`requirements.txt` should be excluded, ie, not part of PR. the version of tf is case-specific

Reply:
true, done